scanners/sslyze.py

```diff
@@ -1,94 +1,91 @@
-import logging
+###
+# Inspect a site's TLS configuration using sslyze.
+#
+# If data exists for a domain from `pshtt`, will check results
+# and only process domains with valid HTTPS, or broken chains.
+#
+# Supported options:
+#
+# --sslyze-serial - If set, will use a synchronous (single-threaded
+#   in-process) scanner. Defaults to true when local, false in cloud.
+# --sslyze-certs - If set, will use the CertificateInfoScanner and
+#   return certificate info. Defaults to true.
```

Comment:
`--sslyze-certs` defaults to True, but can be set to False by passing `--sslyze-certs=false` to disable the use of the certificate info scanner.